templates/collaborative.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Collaborative Movies</title>
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='styles.css') }}" />
+</head>
+<body>
+     <div class="navigation">
+        <a href="{{ url_for('popular') }}">Popular</a>
+        <a href="{{ url_for('content_based_html') }}">Content Based</a>
+        <a href="{{ url_for('collaborative_html') }}">Collaborative</a>
+    </div>
+    <div class="search-results">
+        <h1>Collaborative</h1>
+    </div>
+    <div class="movie-cards" id="movie-cards">
+        {% for movie in movies %}
+            <div class="movie-card">
+                <img class="movie-image" src="{{ movie.image }}" alt="Movie Image">
+                <div class="movie-details">
+                    <h2 class="movie-title">{{ movie.title }}</h2>
+                    <p class="movie-genre">{{ movie.genres | join(', ') }}</p>
+                </div>
+            </div>
+        {% endfor %}
```

Comment:
Indentation error in line 27. The closing 'div' tag should be indented to match the opening 'for' tag.